### PR TITLE
Tech Debt cleanup & refactoring of network service with unit test coverage!

### PR DESCRIPTION
**Pull request by sk-enya**
Cleanup Dead Code:

closes: #599 
closes: #600

### CODE REVIEW

File: src/actions/earnAction.ts [hunk 1]
```diff
@@ -39,53 +49,47 @@ const getEarnInfoSuccess = (
...
+    earnService.loadL2LpInfo(),
   ])
   dispatch(
     getEarnInfoSuccess(
-      L1LPInfo.poolInfo,
-      L1LPInfo.userInfo,
-      L2LPInfo.poolInfo,
-      L2LPInfo.userInfo
+      (L1LPInfo as any).poolInfo,
+      (L1LPInfo as any).userInfo,
```
Comment (wsdt) [inline src/actions/earnAction.ts:58]: typing these things (at least for newer code) might be a high impact task too at some point.

**Review by wsdt — APPROVED** [review]

File: src/actions/earnAction.ts (hunk 1 above)
Comment (sk-enya) [reply src/actions/earnAction.ts:58]: Yup good point, noted will cleanup this while working on redux toolkit with typings changes for state!

**Review by boyuan-chen — APPROVED** [review]
The PR is too long. I will approve it first and test it in the staging env.

**Comment by sk-enya** [comment]
Sure, make sense, there is no as such breaking changes than refactoring and adding more unit test cases! merging PR so we can test on staging and easy to address the issues!

**Pull request merged**